docs/use-cases.rst

```diff
@@ -196,6 +196,93 @@ Two parameters to be configured in this type of implementation are outlined:
++--------------------------------------------------------+--------------------------------------------------------+
+| **XML**                                                | **XML**                                                |
++--------------------------------------------------------+--------------------------------------------------------+
+| .. literalinclude:: ../code/XMLTester.xml              | .. literalinclude:: ../code/XMLTester.xml              |
+|    :language: xml                                      |    :language: xml                                      |
+|    :start-after: <!-->CONF_DS_RDNCY_SCENARIO_SERVER<-->|    :start-after: <!-->CONF_DS_RDNCY_SCENARIO_CLIENT<-->|
+|    :end-before: <!--><-->                              |    :end-before: <!--><-->                              |
++--------------------------------------------------------+--------------------------------------------------------+
+
+.. _discovery_server_persistency_scenario_setup:
+
+UDPv4 persistency example
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+All participants keeps record of all endpoints discovered (other participants, subscribers or publishers). Different
+kind of participants populate this record with different procedures:
+
+- *clients* receive this information from its *servers*.
+- *servers* receive this information from its *clients*.
+
+Given that *servers* used to have many *clients* associated, this is a lengthy process. In case of *server* failure we
+may be interested in speed up this process when the *server* restarts.
+
+Keep the discovery information in a file synchronize with the *server*'s record fulfills the goal. In order to enable
+this we must just specify the :ref:`discovery protocol <discovery_protocol>` as **BACKUP**.
+
+Once the *server* is created it generates a *server-<GUIDPREFIX>.db* (*exempli gratia
+server-73-65-72-76-65-72-63-6C-69-65-6E-74.db*) on its process working directory.
+
+If we want to start anew we must remove the file in order to prevent old discovery info from being loaded.
+
+.. _discovery_server_partitioning_setup:
+
+UDPv4 partitioning using servers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*Server* association can be seen as another isolation mechanism besides :ref:`domains <listening_locators>` and
+:ref:`partitions <partitions>`. *Clients* that do not share a *server* cannot see each other and belong to isolated
+networks. In order to connect server isolated networks we can:
+
+    1. Connect each *client* to both *servers*.
+    2. Connect one *server* to the other.
+    3. Create a new *server* linked to the *servers* the clients are connected to.
+
+Options 1 and 2 can only be implemented by modifying attributes or XML configuration files beforehand. In this regard
+they match the domain and partition strategy. Option 3 can be implemented at runtime, that is, when the isolated
+networks are already up and running.
```

Comment:
I'd provide snippet examples here

Reply:
Ok, The first case is already covered above but I can introduce the others.

Reply:
:+1: You could make a link to the first one then